### PR TITLE
Add a new option - Possible column names

### PR DESCRIPTION
**Pull request by jadamec**
Let's give our end users greater possibilities of importing their tables without useless renaming or reordering columns.

### CODE REVIEW

**Comment by GlennM** [comment]
Thanks for the PR, we'll have a look at it soon!

File: 3.1/imports/model.md [hunk 1]
```diff
@@ -44,6 +44,24 @@ public function model(array $row)
 }
 ```
 
+## Possible column names
+
+In case you want to import rows by several possible names, you can use `??` operator. If the column with the first name (in example _client_name_) exists and is not NULL, return its value; otherwise look for second possible name (in example _client_) etc.
```
Comment (patrickbrouwers) [inline 3.1/imports/model.md]: > several possible names

several possible column names (using `WithHeadingRow`)

> you can use `??` operator

you can use null coalescing operator (`??`).

File: 3.1/imports/model.md [hunk 2]
```diff
@@ -44,6 +44,24 @@ public function model(array $row)
...
+
+```php
+public function model(array $row) {
+  return new Post([
```
Comment (patrickbrouwers) [inline 3.1/imports/model.md]: use `User` to keep consistent with other examples

File: 3.1/imports/model.md [hunk 3]
```diff
@@ -44,6 +44,24 @@ public function model(array $row)
...
+    'name' => $row['client_name'] ?? $row['client'] ?? $row['name'] ?? null
+  ]);
+}
+```
+
+:::warning
```
Comment (patrickbrouwers) [inline 3.1/imports/model.md]: Warning is not needed, this package doesn't support PHP version < 7 anyway

**Comment by patrickbrouwers** [comment]
Thanks @jadamec

**Pull request merged**